app/auth/views/login_utils.py

```diff
@@ -0,0 +1,30 @@
+from flask import session, redirect, url_for
+from flask_login import login_user
+
+from app.config import MFA_USER_ID
+from app.log import LOG
+
+
+def after_login(user, next_url):
+    """
+    Redirect to the correct page after login.
+    If user enables MFA: redirect user to MFA page
+    Otherwise redirect to dashboard page if no next_url
+    """
+    if user.enable_otp:
+        session[MFA_USER_ID] = user.id
+        if next_url:
+            return redirect(url_for("auth.mfa", next_url=next_url))
+        else:
```

Comment:
The `else` clause here is redundant since the previous line has already returned. Of course, this is only the taste on how to organize the code, not the logic itself. May be the `else` clause makes it easier to read. Please feel free to keep this way for the consistency.

Reply:
The `else` is indeed redundant.